packages/cdc-react/src/components/ProfileHeader/ProfileHeaderUserProfileMenuItem/ProfileHeaderUserProfileMenuItem.tsx

```diff
@@ -70,7 +70,16 @@ export const ProfileHeaderUserProfileMenuItem = ({
     }
   };
 
+  const handleEnter = (e: KeyboardEvent) => {
+    if (e.key === "Enter") {
+      const button: HTMLButtonElement | null | EventTarget = e.target;
+      (button as HTMLButtonElement).getElementsByTagName("button")[0]?.click();
+    }
+  };
+
   useEffect(() => {
+    document.body.addEventListener("keydown", handleEnter, true);
```

Comment:
Would it be better to put an `onKeyDown` handler on the `button` element instead of a global body keydown event listener?

Reply:
Yes, I think. Good idea. Updated!